_sass/pages/_find_a_participating_post_office.scss

```diff
@@ -0,0 +1,35 @@
+@use '../variables' as *;
+
+.help-article {
```

Comment:
These styles will apply to every page, not just the Post Office search page, so I don't think `.help-article .usa-collection` is specific enough a selector to avoid potential side-effects on unrelated pages.

I think we'd want to use either a selector specific to this page, or create a separate stylesheet specifically for this page. For the former, you could either create a new wrapping element for the affected content, or maybe use [the existing body class page configuration](https://github.com/18F/identity-site/blob/4fabd2ad891808ce3d486c805ddbbaad1d63489e/_layouts/base.html#L14) ([example](https://github.com/18F/identity-site/blob/4fabd2ad891808ce3d486c805ddbbaad1d63489e/_layouts/help_landing.html#L3-L4)).

Reply:
Alternatively, if we could find a way to style this without ad hoc custom styling and instead using design system utility classes, that might avoid the need for custom styles altogether.

https://designsystem.digital.gov/utilities/

Reply:
> I think we'd want to use either a selector specific to this page, or create a separate stylesheet specifically for this page.

SCSS noob here: AIUI, we are created a separate one, but then we merge it into _index.scss with the `@forward` statement, is that right? Would it be easiest to only include this on the relevant page?

Reply:
I thought my initial selectors would be enough to avoid side-effects. I did not see them on other pages but can see in the future how they could be applied easy.

I added a class inside post-office-search.html and then updated the css file to include additional nesting, see this commit 02bf0a4cd21291605af2c4166d9722ffa5e9cf36.

Reply:
> SCSS noob here: AIUI, we are created a separate one, but then we merge it into _index.scss with the `@forward` statement, is that right?

Yes, this is a separate stylesheet, but it's compiled into the common stylesheet bundle that's loaded on every page, so these styles will be present on every page of the brochure site. 

>Would it be easiest to only include this on the relevant page?

There are some advantages to doing this (size, isolation), but I wouldn't say it's easy, since we'd need to create and load a separate stylesheet bundle for the specific page, and we don't have an existing pattern to draw from in this project to make that easier.

>I added a class inside post-office-search.html and then updated the css file to include additional nesting, see this commit https://github.com/18F/identity-site/commit/02bf0a4cd21291605af2c4166d9722ffa5e9cf36.

That change makes sense to me 👍  We could probably even eliminate or combine the wrapping `.help-article` selector altogether to avoid the excessive nesting, but it's a small point.

Reply:
I eliminated the .help-article selector. I will merge once pipeline finishes. Thanks for the suggestions and comments.